schemas/iso19139/src/main/plugin/iso19139/process/onlinesrc-remove.xsl

```diff
@@ -28,26 +28,47 @@ Stylesheet used to remove a reference to a online resource.
 <xsl:stylesheet xmlns:geonet="http://www.fao.org/geonetwork" xmlns:gmd="http://www.isotc211.org/2005/gmd"
                 xmlns:gco="http://www.isotc211.org/2005/gco"
                 xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:digestUtils="java:org.apache.commons.codec.digest.DigestUtils"
+                xmlns:exslt="http://exslt.org/common"
```

Comment:
```suggestion
                xmlns:exslt="http://exslt.org/common"
                exclude-result-prefixes="#all"
```

Otherwise the namespaces end in the `gmd:distributionInfo` element:

```
<gmd:distributionInfo xmlns:xs="http://www.w3.org/2001/XMLSchema"
                         xmlns:digestUtils="java:org.apache.commons.codec.digest.DigestUtils"
                         xmlns:exslt="http://exslt.org/common">
...
```